src/flask/logging.py

```diff
@@ -72,7 +72,7 @@ def _has_config(logger):
 
 
 def create_logger(app):
-    """Get the the Flask apps's logger and configure it if needed.
+    """Get the Flask apps's logger and configure it if needed.
```

Comment:
Can you change “apps’s” to “app’s” while you’re at it?

Reply:
figured that might have been too many fixes for one PR ;) -- it's changed now.